### ISSUE TITLE
File list: 2 users in the creator column

### ISSUE THREAD

**Issue by funkycram** [humhub/cfiles#161]
If a file has been updated by somebody other than the creator, in the creator column, the user that made this update is displayed in addition to the creator.
It's confusing for users because it looks like there are 2 creators (we can't understand that the 2nd one is the one who updated).
Also, since there are cases (which I couldn't identify) where the file was updated by a user other than the creator for no apparent reason, it's weird to see this second "creator".

So I suggest to remove this code:
https://github.com/humhub/cfiles/blob/685343426847f0d1924496c7f2a7ce65dedc6bb2/widgets/views/fileSystemItem.php#L106-L108

**Comment by luke-** [comment]
@funkycram Yes, this is indeed confusing. I can't think of a case at the moment either.  Would you like to create a PR for this?

**Comment by funkycram** [comment]
@luke-  https://github.com/humhub/cfiles/pull/162

**Comment by luke-** [comment]
@funkycram Thanks!
